app-emulation/docker-credential-helpers/docker-credential-helpers-0.6.0.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 2001-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
You don't need to touch this file at all, we only update the copyright headers for new submits.